src/sidebar/util/test/oauth-client-test.js

```diff
@@ -229,16 +228,14 @@ describe('sidebar/util/oauth-client', () => {
       });
 
       return authorized.then(() => {
-        const params = {
+        const params = new URLSearchParams({
           client_id: config.clientId,
           origin: 'https://client.hypothes.is',
           response_mode: 'web_message',
           response_type: 'code',
           state: 'notrandom',
-        };
-        const expectedAuthUrl = `${config.authorizationEndpoint}?${stringify(
-          params
-        )}`;
+        });
+        const expectedAuthUrl = `${config.authorizationEndpoint}?${params}`;
```

Comment:
Is `${params}` explicit enough that we're stringify-ing a `URLSearchParams` object here?